src/vcs/git.js

```diff
@@ -19,7 +19,11 @@ export async function revision(config, cwd) {
 }
 
 export async function checkoutRevision(config, cwd, repository, ref, rev) {
-  await run(`${config.git} fetch ${repository} ${ref}`, { 
+  await run(`${config.git} remote add dummy ${repository}`, {
```

Comment:
tiny nit here can you make "dummy" a constant and perhaps name it something like tc-vcs-remote or something ?